### DISCUSSION TITLE
Netflow support version and flush_interval paramater

### DISCUSSION THREAD

**Discussion by hunterelectros** [rkosegi/netflow-collector#80] [Q&A] [answered]
Hello!

1) Flush_interval paramater - please explain what it means?
2) Does this exporter only support version 5 of the netflow protocol? If version 9 is not supported, are there plans to add such functionality?

**Comment by rkosegi** [comment, marked as answer]
1) `flush_interval` defines time for which is every metric entry kept alive. Stale entries are removed after given time to prevent memory exhaustion.

2) Only version 5. There are no plans for 9 or IPFIX, but PRs are always welcome
